configure/CMakeLists.txt

```diff
@@ -212,3 +212,5 @@ if(NOT WIN32)
 endif()
 include(GNUInstallDirs)
 include(configure/coveralls.cmake)
+
+option(TANGO_COMPILE_TEST_SUITE "Compile the test suite" ON)
```

Comment:
LGTM but just my 2c: the standard variable for enabling tests in CMake projects is `BUILD_TESTING` an IMO we should follow that. It is defined in [CTest module](https://cmake.org/cmake/help/latest/module/CTest.html) which looks more or less like:

```cmake
option(BUILD_TESTING "Build the testing tree." ON)

if(BUILD_TESTING)
  # ...
  # Make sure testing is enabled
  enable_testing()
  #...
endif()
```

According to [`add_test()` documentation](https://cmake.org/cmake/help/latest/command/add_test.html), it's no-op when `enable_testing()` was not called.

I think it would be enough to remove explicit call to `enable_testing()` from main Cmake config file and pass `-DBUILD_TESTING=OFF` to achieve  what you wanted (cxxtests would still be generated though).

BTW, to skip building tests without any code changes, you could just:
```bash
make tango tango-static
make install/fast
```

Reply:
Sure! It should be `if(BUILD_TESTING)` but I see you changed that in #668. Thanks!